_posts/2015-11-03-payment-integrations.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Payment Integrations"
+date: "2015-11-03 09:00:00"
+icon: money
+categories: tech-stories
+authors: ["Xavier"]
+---
+
+Bank transfers and credit cards aren't the only payment methods available when buying a product from an ePages shop.
+Unlike in the real world, online shopping offers a much wider range of possibilities and solutions such as [Ingenico](http://www.ingenico.com/), [Skrill](https://www.skrill.com/en/) or [Klarna](https://www.klarna.com/us), which can be used to purchase products from our shops.
+
+# General overview
+
+The following procedure describes the process of buying a product:
+
+1. A customer browses an ePages shop and finds a product they want to buy.
+2. The customer adds that product to a basket and starts the check out process.
+3. After choosing payment and shipping methods, the customer is redirected to an external site where they can complete the payment.
+4. Finally, our customer is redirected back to the ePages shop, where they are informed about the status of their order.
+
+That external site mentioned above belongs to a specific payment method - the one that was chosen by the customer from those that are offered in the shop -, which needs to know what is being bought so that payment can be completed.
```

Comment:
That external site mentioned above belongs to a specific payment method - the one that was chosen by the customer from those that are offered in the shop. The Payment method needs to know what is being bought so that payment can be completed.